### PR TITLE
Update css-scrollbar.json

### PR DESCRIPTION
**Pull request by theSdev**
Added info about Firefox supporting W3C editor's draft spec.

### CODE REVIEW

File: features-json/css-scrollbar.json [hunk 1]
```diff
@@ -322,32 +326,33 @@
...
-    "3":"Scrollbar styling doesn\u2019t work in [WKWebView](https://issues.apache.org/jira/browse/CB-10123)"
+    "1":"Only supports styling [scrollbar colors](https://msdn.microsoft.com/en-us/library/ms531155%28v=vs.85%29.aspx), no other properties to define the scrollbar's appearance. (not on standards track)",
+    "2":"Supports scrollbar styling via CSS [pseudo-properties](https://webkit.org/blog/363/styling-scrollbars/). (not on standards track)",
+    "3":"Scrollbar styling doesn\u2019t work in [WKWebView](https://issues.apache.org/jira/browse/CB-10123)",
+    "4":"Can be enabled by setting the layout.css.scrollbar-colors.enabled and layout.css.scrollbar-width.enabled flags to true.",
+    "5":"Firefox 63 supported scrollbar-face-color and scrollbar-track-color seperate properties (now dropped from the spec) instead of unified scrollbar-color property."
   },
-  "usage_perc_y":83.68,
-  "usage_perc_a":2.89,
+  "usage_perc_y":0,
```
Comment (HM100) [inline features-json/css-scrollbar.json]: Please roll it back its value to 83.68

File: features-json/css-scrollbar.json [hunk 2]
```diff
@@ -322,32 +326,33 @@
...
+  "usage_perc_a":86.57,
```
Comment (HM100) [inline features-json/css-scrollbar.json]: Also roll it back to 2.89

File: features-json/css-scrollbar.json [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
   "links":[
     {
-      "url":"https://bugzilla.mozilla.org/show_bug.cgi?id=77790",
+      "url":"https://bugzilla.mozilla.org/show_bug.cgi?id=1460109",
       "title":"Firefox support bug"
     },
+    {
+        "url":"https://webkit.org/blog/363/styling-scrollbars/",
+        "title":"Webkit blog post describing their non-standard support"
+    },
```
Comment (HM100) [inline features-json/css-scrollbar.json]: And remove the comma as well

Comment (theSdev) [reply features-json/css-scrollbar.json]: I genuinely fail to see how this comma is extra.

Comment (HM100) [reply features-json/css-scrollbar.json]: Please apply the fix. Until you do that, it is considered error.

File: features-json/css-scrollbar.json [hunk 4]
```diff
@@ -322,32 +326,33 @@
...
     "and_qq":{
-      "1.2":"y x #2"
+      "1.2":"a x #2"
     },
     "baidu":{
-      "7.12":"y x #2"
+      "7.12":"a x #2"
     }
   },
-  "notes":"Currently scrollbar styling doesn't appear to be on any standards track.",
```
Comment (HM100) [inline features-json/css-scrollbar.json]: And please add some things back

**Comment by theSdev** [comment]
Thank you for helping me along 😊

**Comment by Fyrd** [comment]
Nice work @theSdev & @HM100, thanks!

**Pull request merged**